src/client/entity-editor/common/entity.js

```diff
@@ -19,49 +19,52 @@
 
 // @flow
 
-import Icon from 'react-fontawesome';
+// import Icon from 'react-fontawesome';
 import React from 'react';
 import {genEntityIconHTMLElement} from '../../helpers/entity';
 
 
 type EntityProps = {
 	disambiguation?: ?string,
-	link?: string | false,
 	text: string,
 	type: string,
 	unnamedText?: string
 };
 
 function Entity(
-	{disambiguation, link, text, type, unnamedText}: EntityProps
+	{disambiguation, id, text, type, unnamedText}: EntityProps
 ) {
-	const nameComponent = text || <i>{unnamedText}</i>;
+	const url = type === 'Area' ?
+		`//musicbrainz.org/area/${id}` :
```

Comment:
Are you sure this link will work? I can't seem to find a way to test it.

Reply:
I have taken this from here https://github.com/bookbrainz/bookbrainz-site/blob/4498bddf016efb0b96fbd4f56a9e4af9c7a56f43/src/client/components/pages/parts/search-results.js#L109